packages/components/src/DateTimePickers/DateTimeRange/Manager/Manager.component.js

```diff
@@ -20,11 +20,8 @@ function DateTimeRangeManager(props) {
 	const [state, setState] = useState(initialState);
 
 	useEffect(() => {
-		if (
-			!isEqual(state.startDateTime, props.startDateTime) ||
-			!isEqual(state.endDateTime, props.endDateTime)
-		) {
-			setState({ startDateTime: props.startDateTime, endDateTime: props.endDateTime });
+		if (!isEqual(state.startDateTime, startDateTime) || !isEqual(state.endDateTime, endDateTime)) {
+			setState({ startDateTime, endDateTime });
```

Comment:
`startDateTime` and `endDateTime` are destructured from props, no need to use `props.startDateTime`